README.md

```diff
@@ -1 +1,146 @@
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 ["webpack Todo-list project of Microverse] <a name="about-project"></a>
+
+> This project is about displaying todo list from array object
+
+**[webpack Todo-list project of Microverse]** is a great project for beginners to learn how to build a simple use objects
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- **HTML**
+- **CSS**
+- **JS**
+- **webpack**
+- **linters **
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Using Linters**
+- **objects for dynamic manipulation**
+-
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Install [Node Js](https://nodejs.org/en) and [NPM](https://docs.npmjs.com/cli/v6/commands/npm-install) and [Git](https://git-scm.com/downloads) on your local computer
+
+> Please first check this video most of your questions answer are here : [video link]
+
+### Install
+
+pin your commandline (if your are on windows ) or Terminal (if you are on Mac or Linux) and paste this command:
+
+```sh
+git clone https://github.com/ghermaico135/webpack-todo.git
+```
+
+then you will see the **webpack-todo** on your system then type this:
+
+```sh
+cd webpack-todo
+```
+
+Now you successfully installed it.
+
+### Usage
+
+Inside **webpack-todo** you can install the node and run npm start open it inside your browser.
+
+### Run tests
+
+<!-- To run tests, run the following command: -->
+
+### Deployment
+
+You can deploy this project using [Netlify](https://www.netlify.com/) or [Github Pages]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Michael okbamichael**
+
+- GitHub: [ghermaico135](https://github.com/ghermaico135)
+- Twitter: [@Mikidarling3](https://twitter.com/@Mikidarling3)
+- LinkedIn: [Michael okbamichael](https://www.linkedin.com/in/michael-okbamichael-966106263/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Adding and removing form list dynamically**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page] (https://github.com/ghermaico135/webpack-todo/issue).
```

Comment:
- [ ] your issue link is broken. please, kindly use a working one to keep your README professional.